scripts/release.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
```

Comment:
You can see more about this script on the [watch](https://github.com/mikeal/watch/pull/104) repo.  We also use it at my day job for all our apps and libraries, via our [`ta-scripts`](https://github.com/TechnologyAdvice/ta-scripts) package.

So, it's battle tested and works great.